docs/tutorials/multimodal/advanced_topics/customization.ipynb

```diff
@@ -1188,6 +1188,88 @@
    "cell_type": "markdown",
    "id": "db39fec1",
    "metadata": {},
+   "source": [
+    "### model.mmdet_image.checkpoint_name\n",
+    "\n",
+    "Specify a MMDetection model supported by [MMDetection](https://mmdetection.readthedocs.io/en/latest/user_guides/inference.html). Please use \"yolox_nano\", \"yolox_tiny\", \"yolox_s\", \"yolox_m\", \"yolox_l\", or \"yolox_x\" to run our modified YOLOX models that are compatible to Autogluon."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf07cc08",
+   "metadata": {},
+   "source": [
+    "```\n",
+    "# default used by AutoMM\n",
+    "predictor = MultiModalPredictor(hyperparameters={\"model.mmdet_image.checkpoint_name\": \"yolov3_mobilenetv2_8xb24-320-300e_coco\"})\n",
+    "# choose YOLOX-L\n",
+    "predictor = MultiModalPredictor(hyperparameters={\"model.mmdet_image.checkpoint_name\": \"yolox_l\"})\n",
+    "# choose DINO-SwinL\n",
+    "predictor = MultiModalPredictor(hyperparameters={\"model.mmdet_image.checkpoint_name\": \"dino-5scale_swin-l_8xb2-36e_coco\"})\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64f465e9",
+   "metadata": {},
+   "source": [
+    "### model.mmdet_image.output_bbox_format\n",
+    "\n",
+    "The output bounding box format:\n",
+    "\n",
+    "- `\"xyxy\"`: Output [x1,y1,x2,y2]. Bounding boxes are represented via corners, x1, y1 being top left and x2, y2 being bottom right. This is our default output format.\n",
+    "- `\"xywh\"`: Output [x1,y1,w,h]. Bounding boxes are represented via corner, width and height, x1, y1 being top left, w, h being width and height."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "87be5d56",
+   "metadata": {},
+   "source": [
+    "```\n",
+    "# default used by AutoMM\n",
+    "predictor = MultiModalPredictor(hyperparameters={\"model.mmdet_image.output_bbox_format\": \"xyxy\"})\n",
+    "# choose xywh output format\n",
+    "predictor = MultiModalPredictor(hyperparameters={\"model.mmdet_image.output_bbox_format\": \"xywh\"})\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "30c7ec4d",
+   "metadata": {},
+   "source": [
+    "### model.mmdet_image.frozen_layers\n",
+    "\n",
+    "The layers to be frozen. All layers that contain such substring will be frozen."
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
The format seems wrong. See the tutorial: http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-3737/9382008/tutorials/multimodal/advanced_topics/customization.html#model-mmdet-image-frozen-layers